rpcclient/errors.go

```diff
@@ -338,6 +338,17 @@ func (r BitcoindRPCErr) Error() string {
 	return "unknown error"
 }
 
+// BitcoindErrMap is a map of additional errors bitcoind can throw that are
+// version dependent (e.g. versions up to v29 return the error as specified in
+// `Error()` above, while versions v30 and beyond return the error as mapped
+// here.
+var BitcoindErrMap = map[string]error{
```

Comment:
Why don't we just introduce 2 new errors instead of mapping it to the old one ?

Reply:
Because the errors were simply renamed, I wanted to keep the same semantic meaning (e.g. the same error constant) in case we matched on them anywhere. We currently don't, so we could've added new errors. But I still find it slightly better in terms of understanding what's going on. Added some additional context to the comment of the new map, why we use it and when we should create new constants.
I hope that makes sense to you?